tools/cldr-apps/src/main/java/org/unicode/cldr/web/SearchManager.java

```diff
@@ -211,6 +212,14 @@ private void addCodes(final String locale) {
             logger.finest(() -> "AddCodes made resolved " + locale);
 
             for (final String x : resolvedFile.fullIterable()) {
+                if (x.startsWith("//ldml/annotations/annotation")) {
+                    final String cp = AnnotationUtil.removeEmojiVariationSelector(request.value);
+                    if (x.contains("[@cp=\"" + cp + "\"]") && x.contains("[@type=\"tts\"]")) {
+                        response.addResult(new SearchResult(x, "tts: " + cp, locale));
+                    }
+                    continue; // Do not try to match otherwise code for annotation
```

Comment:
This prevents the  `X -keywords` "code" from  path header from matching as a substring.